src/configuration.ts

```diff
@@ -0,0 +1,89 @@
+import { Environment, FormatEnum, PrecomputedFlag } from './interfaces';
+import { generateSalt, obfuscatePrecomputedFlags, Salt } from './obfuscation';
+import { Attributes, ContextAttributes } from './types';
+
+export interface PrecomputedConfigurationResponse {
```

Comment:
Please export this in the `index.ts` file as well. I will need this here after I call `JSON.parse` https://github.com/Eppo-exp/js-client-sdk/pull/117/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R659